src/app/downloads/downloads-table.component.html

```diff
@@ -34,22 +33,20 @@
           </td>
           <td>{{podcastTableData.releaseDate}}</td>
           <td>{{podcastTableData.totalForPeriod | largeNumber}}</td>
-          <td>{{podcastTableData.avgPerIntervalForPeriod | largeNumber}}</td>
           <td *ngIf="podcastTableData.allTimeDownloads !== undefined">{{podcastTableData.allTimeDownloads | largeNumber}}</td>
           <td *ngIf="podcastTableData.allTimeDownloads === undefined"></td>
         </tr>
         <tr *ngFor="let episode of episodeTableData">
           <td>
             <prx-checkbox *ngIf="showEpisodeToggles; else episodeTitle"
               small [checked]="episode.charted" [color]="episode.color"
-              (change)="toggleChartEpisode(episode, $event)">{{episode.title}}</prx-checkbox>
+                          (change)="toggleChartEpisode(episode, $event)"><span [title]="episode.title">{{episode.title}}</span></prx-checkbox>
```

Comment:
#214 it was said that it's ok to include the title even when the title is not truncated

Reply:
👍